app/_src/gateway/admin-api/index.md

```diff
@@ -1119,6 +1353,7 @@ HTTP 200 OK
 
 
 ---
+{% endunless %}
```

Comment:
Comment re next block - Retrieve Plugin Schema. The "DB-Less Compatible" badge doesn't make sense in the context of Konnect.

Reply:
Also we have a few extra horizontal lines in the doc

![Screen Shot 2022-12-19 at 3 56 04 PM](https://user-images.githubusercontent.com/2746479/208550572-4d35b8d5-405d-45bf-85db-e8fe0f5b56aa.png)